packages/forms/src/Form.test.js

```diff
@@ -373,3 +375,17 @@ describe('<Form/>', () => {
 		});
 	});
 });
+
+describe('<UIForm/>', () => {
```

Comment:
No need for another `describe` because you're still testing `<Form>`. Please move the test in the previous `describe`.